PLAN.md

```
@@ -1,179 +1,216 @@
-Spring DB에 `is_required`, `max_select` 컬럼이 있음. 현재 모델에 누락.
+## 1단계: 의존성 및 설정 추가
 
-```python
-class OptionGroup(BaseModel):
-    group_id: int = Field(alias="groupId")
-    group_name: str = Field(alias="groupName")
-    is_required: bool = Field(alias="isRequired")
-    max_select: int = Field(alias="maxSelect")
-    options: list[Option] = Field(default_factory=list)
+### `requirements.txt` 추가
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)